qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -200,7 +200,7 @@ class UpgradesTest extends BaseSpecification {
     }
 
     @Tag("Upgrade")
-    @IgnoreIf({ Env.CI_JOB_NAME.contains("postgres") })
+    @IgnoreIf({ Env.CI_JOBNAME.contains("postgres") })
```

Comment:
Are these tests still a problem since the feature flag has been removed and this is enabled with Postgres?  If it still fails, is it because the upgrade tests is using versions of Postgres software prior to categories?